ganttproject/src/main/java/biz/ganttproject/storage/cloud/GPCloudHttpImpl.kt

```diff
@@ -350,23 +370,28 @@ class WebSocketClient {
     }
   }
 
+  fun onXlogReceived(listener: (ObjectNode) -> Unit): () -> Unit {
+    xlogReceivedListeners.add(listener)
+    return { xlogReceivedListeners.remove(listener) }
+  }
+
   private fun sendHeartbeat() {
-    this.websocket?.let { websocket ->
-      websocket.send("HB")
-    }
+    this.websocket?.send("HB")
```

Comment:
Thanks for removing these useless constructions.

Reply:
My pleasure :)